openlibrary/admin/templates/admin/index.html

```diff
@@ -207,25 +207,5 @@ <h3>Unique IPs</h3>
             <div id="overview" style="margin-left:50px;margin-top:20px;width:400px;height:50px"></div>
```

Comment:
Any chance we can deal w/ this inline style while we're here? :)

Reply:
I'm aware of this inline style, but am unable to test it as "unique ips" doesn't seem to be replicable in docker (I suspect because I need admin rights which do not work locally?)

As well as being able to replicate locally, moving this out of an inline style is going to require documenting the component on the [Pattern Library](https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library) or finding a close alternative.

Since the page in current form is broken and throwing JS error, do you mind if we cut open a new ticket for this and handle it separately?

Reply:
sure things, thanks for taking the time time explain